partition/mbr/partition_internal_test.go

```diff
@@ -277,7 +277,7 @@ func TestWriteContents(t *testing.T) {
 			EndHead:       0,
 			EndSector:     2,
 			EndCylinder:   0,
-			Start:         partitionStart,
+			Start:         partitionStart / 512,
```

Comment:
Given that `partitionStart` is only ever used is an entry in `Partition.Start`, and in all cases it should be sector and not bytes (i.e. `/512`), it makes more sense to just change the definition of `partitionStart` from 

```go
const (
        partitionStart   = 2048
)
```

to

```go
const (
        partitionStart   = 2048 / 512
)
```

and then keep `partitionStart` unchanged everywhere it is consumed.

Reply:
I started down the track, and have a number of test failures. The first one I've figured out suggests that 
```
const (
	mbrPartitionFile = "./testdata/mbr_partition.dat"
)
``` 
needs to be recreated, if that makes sense?

Reply:
Should it need to be recreated? We are using the same parameters.

Reply:
Hmm, I see them too, but not sure why. Thinking...